pkg/file/buffer_test.go

```diff
@@ -194,23 +179,20 @@ func TestCopyBuffer(t *testing.T) {
 			for {
 				select {
 				case c := <-sizeC:
+					d := <-dataC
```

Comment:
There is a case in the reader if it encounters an EOF on line 209, only size will be sent. The data channel will not be written and this naked read will block. Is this not a problem?

Reply:
This wasn't the problem because we would never encounter EOF, because test would end as soon as expected data is received. If we attempted to read once more time after receiving expected data, we would encounter this issue. 

However, in latest commit data and size channels have been combined to single channel (there was no need for two channels really).

Reply:
There is one more commit after this in which we will read until EOF and assert expected data only then. 

It make more sense to do it like this because it covers one more case - what if there was more data to read from chunk pipe but we ended earlier.